kirzharov/mlops-demo
# Feature/Add dynamic routing

## Updates

### Inference API
- switch to dynamic routing for models
- add example model configs
- add step to check model schema before inference
- add health check to `inference_api` container
- update example endpoints with dynamic routing
- update section for working with models

### Models
- rename `dataset_structure` param
- fix type for `transform_func`
- remove `transform_func` from batch postprocessing

### Common parts
- move transformations to a package for reusability
- move helpers to `common_utils`
- rename and refactor `check_dataset_structure` func

### Example notebooks
- update dataset schema
- change order for prediction mapping

### Other improvements & fixes
- format Dockerfile
- change default path for service account file
- fix typo for service account file
